org.mixedrealitytoolkit.input/InteractionModes/InteractionModeManager.cs

```diff
@@ -249,7 +258,16 @@ public void UnregisterInteractor(XRBaseInteractor interactor)
             GameObject controllerObject = null;
             if (interactor is XRBaseInputInteractor controllerInteractor)
             {
-                controllerObject = controllerInteractor.xrController.gameObject;
+#pragma warning disable CS0618 // xrController is obsolete
+                if (controllerInteractor.forceDeprecatedInput)
+                {
+                    controllerObject = controllerInteractor.xrController.gameObject;
+                }
+#pragma warning restore CS0618 // xrController is obsolete
+                else
+                {
+                    controllerObject = controllerInteractor.transform.parent.gameObject;
```

Comment:
Same here.

Reply:
@whebertML I'll be closing this PR because a [new one](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/824) was created since several changes were made while retaking this and it was easier to re-do the changes in the [new PR](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/824).  Your feedback was attended in the new PR btw, thank you for providing the feedback.